packages/perspective-jupyterlab/test/jupyter/widget.spec.js

```diff
@@ -9,30 +9,30 @@
 
 const path = require("path");
 const utils = require("@finos/perspective-test");
-const {execute_all_cells} = require("./utils");
-
-const default_body = async (page) => {
-    await execute_all_cells(page);
-    const viewer = await page.waitForSelector(
-        ".jp-OutputArea-output perspective-viewer",
-        {visible: true, timeout: 30000}
-    );
-    await viewer.evaluate(async (viewer) => await viewer.flush());
-    return viewer;
-};
+const {
+    default_body,
+    add_and_execute_cell,
+    assert_no_error_in_cell,
+} = require("./utils");
 
 jest.setTimeout(30000);
 
+const getEditable = async (viewer) => {
+    return await viewer.evaluate(async (viewer) => {
+        return viewer.children[0]._is_edit_mode;
+    });
+};
```

Comment:
Why are we checking this internal (and arbitrary_ property instead of `(await viewer.restore()).plugin_config.editable`?

Reply:
Sounds reasonable